app/presenters/node_info_presenter.rb

```diff
@@ -100,10 +100,12 @@ def halfyear_users
   end
 
   def local_posts
-    @local_posts ||= Post.where(type: "StatusMessage")
-                         .joins(:author)
-                         .where("owner_id IS NOT null")
-                         .count
+    Rails.cache.fetch("NodeInfoPresenter/local_posts", expires_in: 1.hours) do
+      @local_posts ||= Post.where(type: "StatusMessage")
+                          .joins(:author)
+                          .where("owner_id IS NOT null")
```

Comment:
- Layout/MultilineMethodCallIndentation: Align `.where` with `.where` on line 104.
- Rails/WhereNot: Use `where.not(owner_id: nil)` instead of manually constructing negated SQL in `where`.